tests/testapp/test_helpers.py

```diff
@@ -15,35 +15,37 @@ def __new__(cls, id_):
         return super(WithId, cls).__new__(cls, id_, 'type name')
 
 
-def run_testcase_with_django_runner(
```

Comment:
@belugame i'll be more precise next time - what I meant is that your points make sense, so let's leave it as you've done it. Apologies.

For the future, when restoring/undoing things, please try to minimize the area of the diff. E.g.: when undoing some change, try to make it so that line breaks/variable names don't cause 'diff noise'.

But for this MR, don't worry about it, i'll merge it, then make some minor doc improvements with regards to `--no-report` (explain that existing test command options are still valid, plus explicitly add a `djpt` prefix so it won't clash with externals), and then release it to pypi.

Thanks again!